app/app.cpp

```diff
@@ -1,11 +1,46 @@
 #include "app.h"
 #include <format>
+#include <sstream>
+#include <iomanip>
+
+using namespace std::string_literals;
 
 namespace vsite::oop::v1
 {
-/*
-* function implementations
-*/
+	std::string to_hex(unsigned int n) {
+        char s[64];
+        sprintf_s(s, "%X", n);
+        return std::string(s);
+	};
+	std::string to_exp(double n) {
+		std::stringstream ss;
+		ss << std::scientific << std::setprecision(2) << n;
+		return ss.str();
+	};
+    void mult_table(int n, std::stringstream& ss) {
```

Comment:
Ne, ideja je da funkcija prima `std::ostream&` - bilo koji output stream (što onda može biti `std::cout`, ali i `std::stringstream` za unit testove).